### PR TITLE
Update all references of vendor url https://adoptopenjdk.net/ to https://adoptium.net/

### PR DESCRIPTION
**Pull request by himanshu007-creator**
#2589

### CODE REVIEW

**Review by M-Davies — APPROVED** [review]
LGTM. Thanks for this 🙂

**Review by sophia-guo — APPROVED** [review]
LGTM

File: RELEASING.md [hunk 1]
```diff
@@ -148,8 +148,8 @@ Here are the steps:
    - `RELEASE`: "ticked"
    - SUBMIT!!
 8. Once the job completes successfully, check the binaries have uploaded to github at somewhere like https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/jdk8u232-b09
-9. Within 15 minutes the binaries should be available on the website too at e.g. https://adoptopenjdk.net/?variant=openjdk11&jvmVariant=openj9
-10. Since you have 15 minutes free, use that time to update https://github.com/AdoptOpenJDK/openjdk-website/blob/master/src/handlebars/support.handlebars which is the source of  https://adoptopenjdk.net/support.html and (if required) the supported platforms table at https://github.com/AdoptOpenJDK/openjdk-website/blob/master/src/handlebars/supported_platforms.handlebars which is the source of https://adoptopenjdk.net/supported_platforms.html, and also update https://adoptopenjdk.net/release_notes.html ([Sample change](https://github.com/AdoptOpenJDK/openjdk-website/pull/675/commits/563d8e2f0d9d26500a7e8d9eca61b491f73f1f37)
+9. Within 15 minutes the binaries should be available on the website too at e.g. https://adoptium.net/?variant=openjdk11&jvmVariant=openj9
```
Comment (smlambert) [inline RELEASING.md:151]: Please change the example to https://adoptium.net/?variant=openjdk11&jvmVariant=hotspot

Comment (himanshu007-creator) [reply RELEASING.md:151]: okay

**Review by smlambert — APPROVED** [review]
Looks good, thanks @himanshu007-creator !

**Pull request merged**